components/resources/library/src/jsMain/kotlin/org/jetbrains/compose/resources/xmldom/NodeImpl.kt

```diff
@@ -0,0 +1,29 @@
+package org.jetbrains.compose.resources.vector.xmldom
+
+import org.w3c.dom.Node as DomNode
+import org.w3c.dom.Element as DomElement
+
+internal open class NodeImpl(val n: DomNode): Node {
+    override val nodeName: String
+        get() = n.nodeName
+
+    override val localName = "" /* localName is not a Node property, only applies to Elements and Attrs */
+
+    override val namespaceURI = "" /* namespaceURI is not a Node property, only applies to Elements and Attrs */
+
+    override val childNodes: NodeList
+        get() =
```

Comment:
minor: this getter creates a new `NodeList` instance every time it's invoked. Can it reuse the same instance?

Reply:
You got it. Updated to use `by lazy` so the NodeList is cached when used.

I (mostly) copied the original code from NodeImpl in jvmAndAndroidMain which has the same getter issue. Seems out of scope for this commit? Worth changing there as well.